ci/Makefile

```diff
@@ -1,7 +1,7 @@
 HOSTNAME        := gcr.io
 PROJECT         := stargateio
 NAME            := stargate-builder
-VERSION         := v1.0.15
+VERSION         := v1.0.16
```

Comment:
Not sure we need to rev the builder, we can just use v1.0.14?

Reply:
but looks like it's already done so never mind

Reply:
Well, I built new one just in case; wasn't 100% sure, since 3.11 version changed.